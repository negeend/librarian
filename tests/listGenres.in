LIST GENRES
list genres
ADD BOOK books.csv 111111
LIST GENRES
add book books.csv 111118
list genres
add book books.csv 111119
LIST GENRES
ADD COLLECTION books.csv
LIST GENRES
EXIT