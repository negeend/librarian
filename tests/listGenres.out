user: No books in system.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Historical Fiction

user: Successfully added: Harry Potter 1 (J.K. Rowling).

user: Fantasy
Historical Fiction

user: Successfully added: Harry Potter 2 (J.K. Rowling).

user: Fantasy
Historical Fiction

user: 14 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: Ending Library process.
